client/components/domains/transfer-domain-step/transfer-domain-precheck.jsx

```diff
@@ -293,7 +294,24 @@ class TransferDomainPrecheck extends React.Component {
 						isError={ authCodeInvalid }
 					/>
 					{ authCodeInvalid && (
-						<FormInputValidation text={ translate( 'Auth Code invalid' ) } isError />
+						<FormInputValidation
+							text={ translate(
+								'The auth code you entered is invalid. ' +
+									'Please try again, or see {{a}}this support document{{/a}} for more troubleshooting steps.',
```

Comment:
Maybe a better wording would be "Please verify you're entering the correct code, or see this support document ..."